pages/apim/3.x/event-native/apim-event-native-api-management-overview.adoc

```diff
@@ -0,0 +1,89 @@
+[[apim-event-native-api-management-overview]]
+= Gravitee Event-native API Management (BETA)
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_event_native_api_management_overview.html
+:page-folder: apim/event-native
+:page-layout: apim3x
+
+[label label-version]#New in version 3.19.0#
+
+== Event-native API Management via the new evented Gateway (BETA)
+
+Gravitee Platform release 3.19.0 has introduced the first version (BETA) of a new type of Gateway and API definition - an evented Gateway that enables event-native API management.
```

Comment:
The new engine has not been built to be event native. It allows even existing V2 api to solve the drawback listed bellow.

Reply:
Addressed in previous commits